src/Console/ViewModelMakeCommand.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Spatie\ViewModels\Console;
+
+use Illuminate\Console\GeneratorCommand;
+use Symfony\Component\Console\Input\InputOption;
+
+class ViewModelMakeCommand extends GeneratorCommand
+{
+    /**
```

Comment:
Let's lose this entire docblock, to make the class a bit lighter. Also remove them for the other properties.

Reply:
done